frontend/src/services/ChoreService.js

```diff
@@ -79,11 +79,7 @@ const ChoreService = {
 
   checkChore: (id) => {
     let config = setConfig(AuthService.getToken());
-    return axios.put(
-      `${process.env.REACT_APP_API_BASE_URL}/chores/${id}/checked/`,
-      {},
-      config
-    );
+    return axios.get(`${process.env.REACT_APP_API_BASE_URL}/chores/checked/${id}/`, config);
```

Comment:
I changed this in my PR on purpose. We're technically not getting anything, but are instead editing a chore, which is why I used the "put" method. Also, I changed the organization of the URL because the hierarchy would tend to go "object type"->"instance"->"verb/action", so this naturally reflects it.

Reply:
Backend actually returns the chore so it can be updated. But since the schedule page is just getting all the chores I think a Put does makes sense.

Reply:
That's fine, except I don't actually do anything with the edited chore. Either way, that would be expected behavior for a put request anyways.